docs/quickstarts/ruby.md

```diff
@@ -2,3 +2,118 @@
 ---
 
 # Ruby
+
+Deploying your Ruby app on FL0 is easy. Follow this guide to configure your codebase correctly.
+
+## Built-in Language Support
+
+You can build and deploy a Ruby application without a Dockerfile using FL0's built-in automatic builds.
+
+### Specifying a Ruby Version
+
+FL0's build system will attempt to automatically detect the correct version of Ruby to install. We support the following versions of Ruby:
```

Comment:
@carmen-fl0 how does it auto-detect the version? How can the user control which version is used?

Reply:
Ahh great shout - lemme edit this!